docs/install/install-jetson.md

```diff
@@ -0,0 +1,272 @@
+</dependency>
+```
+
+Refer to the [Java setup](https://mxnet.incubator.apache.org/versions/master/install/java_setup.html) page for further information.
+-->
+
+## Use a Pre-compiled MXNet Binary
+
+If you want to just use the pre-compiled binary you can download it from S3:
+* https://s3.us-east-2.amazonaws.com/mxnet-public/install/jetson/1.4.1/libmxnet.so
+
+Place this file in `$MXNET_HOME/lib`.
+
+To use this with the MXNet Python binding, you must match the source directory's checked out version with the binary's source version, then install it with pip.
+
+```bash
+cd $MXNET_HOME
+git checkout v1.4.x
+git submodule update --init
+cd python
+sudo pip install -e .
+```
+
+## Build MXNet from Source
+
+Installing MXNet from source is a two-step process:
+
+1. Build the shared library from the MXNet C++ source code.
+2. Install the supported language-specific packages for MXNet.
+
+You can use a Docker method or you can build from source manually.
+
+### Docker
+
+You must have installed Docker and be able to run `docker` without `sudo`.
+Follow these [setup instructions to get to this point](https://docs.docker.com/install/linux/#manage-docker-as-a-non-root-user).
+Then run the following to execute cross-compilation via Docker.
+
+```bash
+$MXNET_HOME/ci/build.py -p jetson
+```
+
+### Manual
+
+**Step 1** Build the Shared Library
+
+(Skip this sub-step for compiling on the Jetson device directly.)
+Edit the Makefile to install the MXNet with CUDA bindings to leverage the GPU on the Jetson:
+
+```bash
+cp $MXNET_HOME/make/crosscompile.jetson.mk config.mk
+```
+
+Edit the Mshadow Makefile to ensure MXNet builds with Pascal's hardware level low precision acceleration by editing `3rdparty/mshadow/make/mshadow.mk`.
+The last line has `MSHADOW_USE_PASCAL` set to `0`. Change this to `1` to enable it.
+
+```bash
+MSHADOW_CFLAGS += -DMSHADOW_USE_PASCAL=1
+```
+
+Now you can build the complete MXNet library with the following command:
+
+```bash
+cd $MXNET_HOME
+make -j $(nproc)
+```
+
+Executing this command creates a file called `libmxnet.so` in the `mxnet/lib` directory.
+
+**Step 2** Install MXNet Python Bindings (optional)
+
+To install Python bindings run the following commands in the MXNet directory:
+
+```bash
+cd $MXNET_HOME/python
+sudo pip install -e .
+```
+
+Note that the `-e` flag is optional. It is equivalent to `--editable` and means that if you edit the source files, these changes will be reflected in the package installed.
+
+**Step 3** Install the MXNet Java & Scala Bindings (optional)
+
+Change directories to `scala-package` and run `mvn install`.
+
+```bash
+cd $MXNET_HOME/scala-package
+mvn install
+```
+
+This creates the required `.jar` file to use in your Java or Scala projects.
+
+## Conclusion and Next Steps
+
+You are now ready to run MXNet on your NVIDIA Jetson TX2 or Nano device.
+You can verify your MXNet Python installation with the following:
+
+```python
+import mxnet
+mxnet.__version__
```

Comment:
usually it's good to make 2 tensors on the gpu and add them (to make sure CUDA related stuff was built correctly)

Reply:
Good idea! I'll add that in. I was able to verify this myself.